src/main/java/org/edgexfoundry/messaging/ZeroMQEventSubscriber.java

```diff
@@ -58,37 +61,37 @@
 
 	private ZMQ.Socket subscriber;
 	private ZMQ.Context context;
+	private ObjectMapper mapper = new ObjectMapper();
 
 	{
 		context = ZMQ.context(1);
 	}
 
 	public void receive() {
 		getSubscriber();
-		String exportString = null;
-		byte[] exportBytes = null;
-		Event event;
+		JsonNode node;
+		String msgEnvenlope = null;
 		logger.info("Watching for new exported Event messages...");
 		try {
 			while (!Thread.currentThread().isInterrupted()) {
-				if (exportClient) {
-					if (serializedJava) { // supporting legacy Java export distro that shipped Serialized eventString
-						exportString = subscriber.recvStr();
-						event = toEvent(exportString);
-					} else { // supporting new Go export distro that ships JSON
-						exportBytes = subscriber.recv();
-						event = toEvent(exportBytes);
+				msgEnvenlope = subscriber.recvStr();
+				if (!"events".equals(msgEnvenlope)) { // why is every other event "events"?
```

Comment:
When connected directly to Core Data the new messaging is sending multiple part messages [<topic>,<payload>],  if you are not handling multi part messages you will be receive the topic, "events", as a message. I think this logic works, just change the comment to state something like "skip the 'events' topic"

Reply:
Ugh - didn't even think about multi-part Lenny.  Couldn't figure out why the "events" message kept coming in.  I have address the message as multipart in the updated PR.